_topic_map.yml

```diff
@@ -1362,16 +1364,14 @@ Topics:
   Topics:
   - Name: Getting started with Helm on OpenShift Container Platform
     File: getting-started-with-helm-on-openshift-container-platform
-#- Name: OpenShift Pipelines CLI (tkn)
-#  Dir: tkn_cli
-#  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#  Topics:
-#  - Name: Installing tkn
-#    File: installing-tkn
-#  - Name: Basic tkn commands
-#    File: op-tkn-reference
-- Name: Knative CLI (kn) for use with OpenShift Serverless
-  File: kn-cli-tools
```

Comment:
Hey @Preeticp - this removed the kn-cli-tools file from the build. Was this intentional?

@abrennan89 ^

Reply:
Not intentional at all @vikram-redhat, I am not sure how this happened. Apologies @abrennan89.
Vikram, what would be the best way to fix this? Should I create a PR for adding this back again?

Reply:
@preethit yes it's OCP 4.3, 4.4 and 4.5. Please do be careful modifying this, as I've already had to raise PRs to add it back in 2 weeks ago, when it was removed in the same way: https://github.com/openshift/openshift-docs/pull/22670

Reply:
@abrennan89 do you think there is a particular reason that it keeps getting removed? Cause AFAIK, I did not touch it!!
Nevertheless I will fix it.

Reply:
Maybe it would be helpful to move it above the tkn docs, just so if you're doing anything with that whole section it won't get included by mistake? I just didn't want to bump you down the list without asking when I added it, so I stuck it in at the end of the section :slightly_smiling_face: 

It's not the end of the world since we have other CLI docs anyway, but the last time it disappeared I had someone in eng asking where it was and it took me a little while to find out why it had been removed, where, and in what versions.